en/Plugins/Canvas.md

```diff
@@ -0,0 +1,229 @@
+Canvas is a tool for visual note-taking. Lay out notes and other resources on an infinite canvas and connect them with lines.
+
+Visual note-taking allows you to use visual aspects, such as size and position, to make sense of your notes. Connect notes with lines and group related notes together to better understand the relationship between them.
+
+> [!info] Only for Obsidian desktop
+> Canvas is only available in Obsidian for desktop.
+
+## Create a new canvas
+
+To start using Canvas, you first need to create a file to hold your canvas. You can create a new canvas using the following methods.
+
+**Ribbon:**
+
+- In the ribbon on the left, select **Create new canvas** (four rectangles icon) to create a canvas in the same folder as the active file.
+
+**Command palette:**
+
+1. Open the [[Command palette]].
+1. Select **Canvas: Create new canvas** to create a canvas in the same folder as the active file.
+
+**File explorer:**
+
+- In the [[File explorer]], right-click the folder you want to create the canvas in.
+- Select **New canvas**.
+
+> [!note]  The .canvas file extension
+> Obsidian stores the configuration for each canvas in a custom JSON format in a file with the `.canvas` extension.
+
+## Adding cards
+
+You can drag files into your canvas from Obsidian or from other applications. For example, Markdown files, images, audio, PDFs, or even unrecognized file types.
+
+### Add text cards
+
+You can add text-only cards that don't reference a file.
+
+To add a new text card to your canvas:
+
+- Select or drag the blank file icon at the bottom of the canvas.
+
+You can also add text cards by double-clicking on the canvas.
+
+To convert a text card to a file:
+
+1. Right-click the text card and then select **Convert to file...**.
+1. Enter the note name and then select **Save**.
+
+### Add cards from notes
+
+To add a note from your vault to your canvas:
+
+1. Select or drag the document icon at the bottom of the canvas.
+1. Select the note you want to add.
+
+You can also add notes from the canvas context menu:
```

Comment:
We should also mention here that you can drag files from the File Explorer to the canvas to add them.